build.gradle

```diff
@@ -2,13 +2,13 @@ plugins {
 	id "idea"
 	id "java"
 
-	id "com.diffplug.spotless" version "5.7.0"
+	id "com.diffplug.spotless" version "5.8.2"
 	id "com.github.ben-manes.versions" version "0.36.0"
-	id "com.google.cloud.tools.jib" version "2.6.0"
+	id "com.google.cloud.tools.jib" version "2.7.0"
 	id "com.palantir.git-version" version "0.12.3"
 	id "io.spring.dependency-management" version "1.0.10.RELEASE"
 	id "org.hidetake.swagger.generator" version "2.18.2"
-	id "org.springframework.boot" version "2.3.5.RELEASE"
+	id "org.springframework.boot" version "2.4.0"
```

Comment:
have you reviewed the 2.4 release notes? There's some stuff in there around config loading, k8s config integration, and docker image publishing that could be interesting and/or important.

Reply:
I don't think any of the things other than the config include change is relevant.